twitter/twitter_records.bal

```diff
@@ -0,0 +1,703 @@
+# + retweeted_status - Users can amplify the broadcast of Tweets authored by other users by retweeting . Retweets
+#                      can be distinguished from typical Tweets by the existence of a retweeted_status attribute.
+#                      This attribute contains a representation of the original Tweet that was retweeted.
+# + quoted_status_id - This field only surfaces when the Tweet is a quote Tweet. This field contains the integer value
+#                      Tweet ID of the quoted Tweet.
+# + quoted_status_id_str - This field only surfaces when the Tweet is a quote Tweet. This is the string representation
+#                      Tweet ID of the quoted Tweet.
+# + quoted_status - This field only surfaces when the Tweet is a quote Tweet. This attribute contains the Tweet object
+#                   of the original Tweet that was quoted.
+# + quote_count - Indicates approximately how many times this Tweet has been quoted by Twitter users.
+# + reply_count - Number of times this Tweet has been replied to.
+# + entities - Entities which have been parsed out of the text of the Tweet.
+# + extended_entities - When between one and four native photos or one video or one animated GIF are in Tweet,
+#                       contains an array 'media' metadata.
+# + filter_level - Indicates the maximum value of the filter_level parameter which may be used and still stream this
+#                  Tweet. So a value of medium will be streamed on none, low, and medium streams.
+# + matching_rules - Present in filtered products such as Twitter Search and PowerTrack. Provides the id and tag
+#                    associated with the rule that matched the Tweet.
+# + current_user_retweet - Details the Tweet ID of the user’s own retweet (if existent) of a Tweet.
+# + scopes - A set of key-value pairs indicating the intended contextual delivery of the containing Tweet.
+# + withheld_copyright - When present and set to "true", it indicates that this piece of content has been withheld
+#                        due to a DMCA complaint.
+# + withheld_in_countries - When present, indicates a list of uppercase two-letter country codes this content
+#                           is withheld from.
+# + withheld_scope - When present, indicates whether the content being withheld is the "status" or a "user".
+public type Tweet record {
+    string created_at;
+    int id;
+    string id_str;
+    string text;
+    boolean truncated;
+    TweetMetadata metadata?;
+    string source;
+    int? in_reply_to_status_id;
+    string? in_reply_to_status_id_str;
+    int? in_reply_to_user_id;
+    string? in_reply_to_user_id_str;
+    string? in_reply_to_screen_name;
+    TruncatedUser|User user?;
+    Geo? geo;
+    Place? place;
+    Geo? coordinates;
+    json contributors?;
+    boolean is_quote_status;
+    int retweet_count;
+    int? favorite_count;
+    boolean? favorited;
+    boolean retweeted;
+    boolean? possibly_sensitive?;
```

Comment:
Shall we address this also with the nillable vs optional behavior?
There are few other fields below where marked the field as both nillable and optional.
`boolean**?** possibly_sensitive**?**;`

Reply:
We will get Tweet record response for these remote functions (tweet, replyToTweet, retweet, createFavoriteTweet, getHomeTimelineTweets, getTweetMentions)

For some of the remote function, I got possibly_sensitive: null in the response.
for some of the remote function, I haven't get possibly_sensitive param in the response.

When I defined record in these ways, boolean? possibly_sensitive; or boolean possibly_sensitive?;
I got json to record conversion error for some of the test cases.